packages/commonwealth/client/scripts/models/Thread.ts

```diff
@@ -1,5 +1,6 @@
 import { ProposalType } from 'common-common/src/types';
 import type moment from 'moment';
+import { link } from 'server/models/thread';
```

Comment:
Please do not import files from the server on the client, as we don't want to mix client and server code / imports. Ideally we should recreate the type on the client (or maybe import type will work fine, TBD).